.github/workflows/semanticVersionUpdater.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         git commit -m "Bump version to v$NEW_VERSION" -a
         git tag -a "v$NEW_VERSION" -m "${{ github.event.pull_request.title }}"
     - name: Push changes
-      uses: ad-m/github-push-action@main
+      uses: ad-m/github-push-action@master
```

Comment:
Yeah. It appears that the action ID references the repo the action is defined in. `main` didn't exist; `master` does, because the action's repo uses `master`, not `main`.